File: Python3/프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.py
def solution(num_list):
    list1 = str(0)
    list2 = str(0)
    for i in num_list:
        if i % 2 == 0:
            if list1 != 0:
                list1 += str(i)
            else:
                list1 = i
        else:
            if list2 != 0:
                list2 += str(i)
            else:
                list2 = i
    return (int(list1) + int(list2))
            

File: JavaScript/프로그래머스/0/181928. 이어 붙인 수/이어 붙인 수.js
function solution(num_list) {
    let JJAC='', HOLE = '';
    for(let i = 0; i < num_list.length;i++){
        (num_list[i]%2==0)? JJAC += num_list[i]: HOLE += num_list[i];
    }
    return parseInt(JJAC) + parseInt(HOLE);
}